Protecs/src/test/java/uk/ac/ucl/protecs/objects/diseases/CoronavirusInfectiousBehaviourTesting.java

```diff
@@ -0,0 +1,308 @@
+		// seed a number of the specific node to the run
+		SetFractionInfectionsWithCertainNode(1.0, sim, sim.infectiousFramework.setNodeForTesting("exposed"));
+		// Set up a duration to run the simulation
+		int numDays = 100; 
+		// Run the simulation and record the infectious behaviour nodes reached in this simulation
+		List<String> uniqueNodesInRun = getUniqueNodesInSim(sim, numDays);
+		// we would expect only the exposed, presymptomatic, asymptomatic and susceptible nodes to be present in the run
+		List<String> expectedNodes = Arrays.asList("exposed", "susceptible", "presymptomatic", "asymptomatic");
+		// Make sure than no other nodes are reaching in the simulation
+		Assert.assertTrue(expectedNodes.containsAll(uniqueNodesInRun));
+	}
+	@Test
+	public void TestPresymptomaticEndpoints() {
+		// create a simulation and start
+		WorldBankCovid19Sim sim = helperFunctions.CreateDummySim("src/test/resources/InfectiousBehaviourTestParams.txt", false);
+		sim.start();
+		
+		// Make sure there are no new infections
+		sim.params.infection_beta = 0;
+		// Make everyone have a critical infection
+		SetFractionInfectionsWithCertainNode(1.0, sim, sim.infectiousFramework.setNodeForTesting("presymptomatic"));
+		// Make Sure no one's disease progresses beyond a Mild infection
+		HaltDiseaseProgressionAtStage(sim, "Mild");
+		// Set up a duration to run the simulation
+		int numDays = 100; 
+		// Make sure no one recovers from COVID
+		StopRecoveryHappening(sim);
+		// Run the simulation and record the infectious behaviour nodes activated in this simulation
+		List<String> uniqueNodesInRun = getUniqueNodesInSim(sim, numDays);
+		// we would expect only the presymptomatic and mild node to show up in this run
+		List<String> expectedNodes = Arrays.asList("presymptomatic", "mild_case");
+		// Make sure than no other nodes are reaching in the simulation
+		Assert.assertTrue(expectedNodes.containsAll(uniqueNodesInRun));
+		}
+	@Test
+	public void TestAsymptomaticEndpoints() {
+		// create a simulation and start
+		WorldBankCovid19Sim sim = helperFunctions.CreateDummySim("src/test/resources/InfectiousBehaviourTestParams.txt", false);
+		sim.start();
+		// Make sure there are no new infections
+		sim.params.infection_beta = 0;
+		// Make everyone have a critical infection
+		SetFractionInfectionsWithCertainNode(1.0, sim, sim.infectiousFramework.setNodeForTesting("asymptomatic"));
+		// Set up a duration to run the simulation
+		int numDays = 100; 
+		// Run the simulation and record the infectious behaviour nodes activated in this simulation
+		List<String> uniqueNodesInRun = getUniqueNodesInSim(sim, numDays);
+		// we would expect only the asymptomatic and recovered nodes to show up in this run
+		List<String> expectedNodes = Arrays.asList("asymptomatic", "recovered");
+		// Make sure than no other nodes are reaching in the simulation
+		Assert.assertTrue(expectedNodes.containsAll(uniqueNodesInRun));
+		}
+	
+	@Test
+	public void TestMildEndpoints() {
```

Comment:
confirm that infectious do in fact resolve

Reply:
Created additional tests that do this